cmd/githubgen/go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/mitchellh/copystructure v1.2.0 // indirect
 	github.com/mitchellh/reflectwalk v1.0.2 // indirect
 	go.opentelemetry.io/collector/featuregate v1.12.1-0.20240729082905-fb5b1e6aa550 // indirect
```

Comment:
The stable modules should be set to a released version

Reply:
It was v0.12.0 before, then there are multiple commits bumping the version. The latest is https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/9128a9258fe1fee36f198f97b1e3371fc7b77a93.

I can try revert it back to v0.12.0

Reply:
i'm assuming you mean 1.12 :) in which case yeah that makes sense